commands/command-docs.md

```diff
@@ -6,9 +6,9 @@ You can use the optional _command-name_ argument to specify the names of one or
 The reply includes a map for each returned command.
 The following keys are always present in the reply:
```

Comment:
This heading is not correct anymore. I think we should remove it and merge the two lists of fields (always present and sometimes present).

Reply:
sorry, i was eager to merge it..
please just issue a new PR and merge it.

Reply:
ohh sorry. I did not go through the whole file and didn't see it. I will take care of it later